.travis.yml

```diff
@@ -28,3 +39,9 @@ script:
 
 after_success:
   - coveralls
+  - docker login -e $DOCKER_EMAIL -u $DOCKER_USER -p $DOCKER_PASS
```

Comment:
Sorry, n00b question again: what's the point of building and pushing the image (?) after the tests?